### PR TITLE
feat(executors): Add custom torch vision encoder

### PR DESCRIPTION
**Pull request by JoanFM**
**Issue Link**: This commit does not refer to any issue or feature request

**Description:** This commits adds the possibility to the user to use any custom saved torch models.
So far, torchvision encoders can use pretrained models from pytorch (mobilenet, alexnet , resnet and all their flavours) but it can be interested to be able to use custom models trained for a specific purpose and being able to extract feature maps from any of the layers.

Future work could imply creating a hub example on how to deploy your model as document indexer in Jina.

### CODE REVIEW

**Comment by hanxiao** [comment]
I have fixed part of our CI pipeline so that this PR can at least run unit test.

Unfortunately, due to the limitation of current Github action:
- https://github.community/t5/GitHub-Actions/Make-secrets-available-to-builds-of-forks/td-p/30678/highlight/true
- https://github.com/nyurik/auto_pr_comments_from_forks

`CI/label-pr` and `CLA/cla-check` can not run at all 😢

**Comment by hanxiao** [comment]
recheckcla

**Comment by JoanFM** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by JoanFM** [comment]
recheckcla

**Comment by hanxiao** [comment]
labelpr

**Comment by hanxiao** [comment]
@JoanFM thanks a lot for your contribution ❤️ 

We will tackle the github problem on another day!
